src/status_im/contexts/profile/contact/contact_request/view.cljs

```diff
@@ -28,6 +30,14 @@
                                                                 :text (i18n/label
                                                                        :t/contact-request-was-sent)}]))
                                                [public-key message])]
+    (rn/use-mount
+     (fn []
+       (let [listener (.addListener rn/keyboard
+                                    "keyboardDidHide"
+                                    (fn [_event]
+                                      (when (and platform/android? @input-ref)
+                                        (.blur ^js @input-ref))))]
+         #(.remove ^js listener))))
```

Comment:
I think @flexsurfer expects this `#(.remove ...)` is used in the `use-unmount` hook.

I'd say a regular `use-effect` is better since  it can handle both, @flexsurfer wdyt?

Reply:
use-unmount can't be used here because of listener, so this code is ok